ports/nrf/boards/WRL13990/README.md

```diff
@@ -0,0 +1,91 @@
+# wrl13990 Board Support
+
+The Sparkfun `WRL13990` is a very basic, no thrills, `nRF52832-QFAA` based board. It can be used as a DIL socket and eases the development of a Bluetooth Low Energy or other ARM Cortex-M4 applications.
+
+## Board Pins
+An overview of all available pins is in the table below. Most of the pins can be used for PWM, UART, SPI, I2C, etc. However some pins are special like the ADC and NFC provision. Please refer to the datasheet for additional information:
+
+- When using NFC pins take special care about configuration. See note in [datasheet](http://infocenter.nordicsemi.com/pdf/nRF52832_PS_v1.4.pdf), page 17..18. At reset the pins are set to NFC mode!
+- When UART hardware handshaking is required, select the GPIO's you like and set them accordingly in the `mpconfigboard.h`
+
+
+LABEL | BOARD | CPU | REMARK
+--- | --- | --- | ---
+         |P0 |P0 | Reserved 32kHz XTAL 
```

Comment:
I'm sorry i am nitpicking here... 
I feel the board/cpu columns share a bit redundant information. Does it make sense to document the board (silkscreen label) and the CPU pin as two distinct values if they will always be the same? I'm thinking that it could make sense to maybe combine these two columns. What do you think?